### PR TITLE
Doc: update docker-compose

### PR DESCRIPTION
**Pull request by gbonaspetti**
## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -349,9 +349,9 @@ It is possible to specify several databases or even `all`:
     # Run tests on all databases
     pytest --forked --database=all
 
-### Docker-compose test file
+### Docker compose test file
 
-The [docker-compose.yml](docker-compose.yml) file lists the services needed for testing the Tracim backend. Default environment variables used by the containers are written in the [.env](.env) file next to `docker-compose.yml`.
+The [docker-compose.yml](docker compose.yml) file lists the services needed for testing the Tracim backend. Default environment variables used by the containers are written in the [.env](.env) file next to `docker-compose.yml`.
```
Comment (gbonaspetti) [inline backend/README.md:354]:
```suggestion
The [docker-compose.yml](docker-compose.yml) file lists the services needed for testing the Tracim backend. Default environment variables used by the containers are written in the [.env](.env) file next to `docker-compose.yml`.
```

File: backend/.env [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# Those are default values for docker-compose*.yml files
```
Comment (PhilippeAccorsi) [inline backend/.env:1]: Do not remove `-` here.

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]
A small change.
Need to test on new dev installation if all script and test working correctly.

File: backend/.env [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
-# They can be overidden with -e ENV=value on the docker-compose command line
+# Those are default values for docker compose*.yml files
```
Comment (gbonaspetti) [inline backend/.env:1]:
```suggestion
# Those are default values for docker-compose*.yml files
```

File: concourse/scripts/install_docker [hunk 4]
```diff
@@ -3,21 +3,18 @@ set -e
...
   $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null
 apt-get update
-apt-get install -qy docker-ce docker-ce-cli containerd.io
-
-# Offical docker-compose instructions from https://docs.docker.com/compose/install/
-if [ ! -f /usr/local/bin/docker-compose ]; then
-  curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-  chmod +x /usr/local/bin/docker-compose
-fi
+apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
```
Comment (Quiwy) [inline concourse/scripts/install_docker:17]:
```suggestion
apt-get install -qy docker-ce docker-ce-cli containerd.io docker-compose-plugin
```

File: concourse/scripts/install_docker [hunk 5]
```diff
@@ -3,21 +3,18 @@ set -e
 
 # Official docker instructions from https://docs.docker.com/engine/install/debian/#install-using-the-repository
 apt-get update
-apt-get install -qy apt-transport-https ca-certificates curl gnupg lsb-release
-if [ ! -f /usr/share/keyrings/docker-archive-keyring.gpg ]; then
-  curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
-fi
+apt-get install \
```
Comment (Quiwy) [inline concourse/scripts/install_docker:6]:
```suggestion
apt-get install -qy \
```

**Review by Quiwy — COMMENTED** [review]
For now, the CI is blocked because it asks confirmation for package installation, so we need to accept changes when calling `apt-get`, like it was done [here](https://github.com/tracim/tracim/pull/5801/files#diff-9bde39cdfbde0898c6958301a2e6be457f381c6f32bf34e9d75cde2d74436b31L6)

**Review by PhilippeAccorsi — APPROVED** [review]

**Comment by gbonaspetti** [comment]
Tested on branch by @Quiwy

**Pull request merged**